### PR TITLE
Feat/group field change

### PR DESCRIPTION
**Pull request by NaamanTsur**
This PR changes the behavior for changing counties/desks of investigation groups. Now changing the county/desk of one investigation in a group changes all the investigations in the same group.

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 1]
```diff
@@ -553,84 +555,116 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
...
             confirmButtonColor: theme.palette.primary.main,
             confirmButtonText: 'כן, המשך'
-        }).then((result) => {
-            if (result.isConfirmed) {
-                axios.post('/users/changeCounty', {
-                    epidemiologyNumber: indexedRow.epidemiologyNumber,
-                    updatedCounty: newSelectedCounty.id,
-                }).then(() => timeout(1900).then(() => {
+        });
+
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:573]: delete row

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 2]
```diff
@@ -553,84 +555,116 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
...
+        if (result.isConfirmed) {
+            changeCountyLogger.info(`confirmed changing county in investigation ${indexedRow.epidemiologyNumber}`, Severity.LOW);
+
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:576]: delete row

File: server/src/ClientToDBAPI/UsersRoute/mainRoute.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ import {
     GET_IS_USER_ACTIVE, GET_USER_BY_ID, GET_ACTIVE_GROUP_USERS,
     GET_ALL_LANGUAGES, GET_ALL_SOURCE_ORGANIZATION, GET_ADMINS_OF_COUNTY, GET_USERS_BY_DISTRICT_ID, GET_ALL_USER_TYPES, GET_USERS_BY_COUNTY_ID
 } from '../../DBService/Users/Query';
+import { group } from 'console';
```
Comment (ItayBenMoshe) [inline server/src/ClientToDBAPI/UsersRoute/mainRoute.ts:17]: do you use it?

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 4]
```diff
@@ -553,84 +555,116 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
...
                 confirmButtonColor: theme.palette.primary.main,
                 confirmButtonText: 'כן, המשך',
-            }).then((result) => {
-                if (result.isConfirmed) {
-                    axios.post('/landingPage/changeDesk', {
-                        epidemiologyNumbers: [indexedRow.epidemiologyNumber],
-                        updatedDesk: newSelectedDesk.id,
-                    }).then(() => timeout(1900).then(() => {
+            });
+
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:631]: delete row

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 5]
```diff
@@ -553,84 +555,116 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
...
+            if (result.isConfirmed) {
+                changeDeskLogger.info(`confirmed changing desk in investigation ${indexedRow.epidemiologyNumber}`, Severity.LOW);
+
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:634]: delete row

**Review by nevobonneh — APPROVED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Pull request merged**